app/assets/javascripts/components/Header.js

```diff
@@ -0,0 +1,63 @@
+define(['jquery', 'DoughBaseComponent', 'utilities'], function($, DoughBaseComponent, utilities) {
```

Comment:
The 'utilities' file isn't needed for this component so references to that can be deleted here.

Reply:
44345a3